daprdocs/content/en/developing-applications/building-blocks/actors/actors-overview.md

```diff
@@ -54,6 +54,12 @@ You would use Dapr Workflow when you need to define and orchestrate complex work
 
 ## Features
 
+### Namespaced actors
+
+Dapr supports namespaced actors. For example, with Dapr's multi-tenant placement service, sidecars belonging to a tenant named "Tenant A" won't receive placement information for "Tenant B". This means that multiple tenants can have actor types with the same name. 
```

Comment:
I think that if we're introducing a new term "tenant", we should define what it means, but that can become too broad. Maybe it would be simpler to say something like "with Dapr's multi-tenant placement service, where each tenant has its own namespace, sidecars belonging to a tenant named "Tenant A" won't receive placement information for "Tenant B"...."